internal/resources/aws/global_accelerator_endpoint_group.go

```diff
@@ -61,6 +61,7 @@ var regionCodeMapping = map[string]string{
 	"eu-south-1":      "EU",
 	"eu-west-3":       "EU",
 	"eu-north-1":      "EU",
+	"il-central-1":    "EU",
```

Comment:
Should this be the Middle East? https://aws.amazon.com/about-aws/global-infrastructure/regions_az/?p=ngi&loc=2
```suggestion
	"il-central-1":    "ME",
```

Although I don't know what mapping should be for Cloudfront 🤔

Reply:
AWS groups Israel with Europe for a lot of the pricing (see https://aws.amazon.com/global-accelerator/pricing/). Not sure why - maybe they are using Eurovision geography.